.travis.yml

```diff
@@ -81,6 +70,9 @@ jobs:
 
     - stage: Coverage
       php: 7.2
+      install:
+        - composer require --dev "symfony/messenger:4.2.*" --no-update
```

Comment:
why do we need messenger ?

Reply:
The messenger component is needed to test the integration there, which is conditional on whether the extension is present.